README.md

```diff
@@ -14,7 +14,7 @@ As of 2024-11-07, the supported version of StreamX is 0.0.45.
 - [StreamxClientBuilders](src/Builders/StreamxClientBuilders.php) - with which it's possible to
   create default or customized clients,
 - [StreamxClient](src/StreamxClient.php) - with which it's possible to create publishers
-- [Publisher](src/Publisher/Publisher.php) - with which it's possible to make actual publications
+- [Publisher](src/Publisher/Publisher.php) - with which it's possible to make actual ingestions (publishing and unpublishing)
```

Comment:
Now the client is not only for publishing (and unpublishing).
Now it is also able to retrieve channel schema.

So, time to remove the old "publications" wording - replace it with ingesting